docs/Release-Process.md

```diff
@@ -124,9 +124,9 @@ instability and all these versions where tested in the release candidate testing
 For the cloud-api-adaptor we need to wait until the Kata Containers release tag has been created and the
```

Comment:
Ohh here it is one... I would suggest squashing this commit with the previous one.